package.json

```diff
@@ -44,5 +53,10 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "jest": {
+    "transformIgnorePatterns": [
+      "node_modules/(?!@ngrx|(?!deck.gl)|ng-dynamic)"
```

Comment:
ngrx? is for angular also ng-fynamic.. what is deck.gl? do we need it?

Reply:
not sure, just regex I found on web to allow the esm in jest tests